source/data_at_rest_encryption.rst

```diff
@@ -8,6 +8,10 @@ Data at rest encryption for the WiredTiger storage engine in |mongodb| was
 introduced in |mongodb-enterprise| version 3.2. to ensure that encrypted data
 files can be decrypted and read by parties with the decryption key.
 
+Note that data at rest encryption is enabled when starting |PSMDB| providing all
+essential encryption options. You cannot enable or disable encryption while the
+|PSMDB| server is already running.
```

Comment:
Not only that, you cannot change the options even if you stop the server - eg. if it was started with some encryption options it should always be started the same, so even if you stop it you cannot switch from encrypted->non-encrypted or vice versa.